src/OnFail/Action/SkipAction.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\OnFail\Action;
+
+use Psr\Log\LoggerInterface;
+
+/**
+ * Class for skipped actions.
+ * Logs the information about skipped tests.
+ */
+class SkipAction implements ActionInterface
```

Comment:
What's the purpose of this?

Reply:
If developers can add some custom actions, they should also be able to skip some actions.